src/pkg/cli/client/provider.go

```diff
@@ -135,8 +135,10 @@ type Provider interface {
 
 type AccountInfo interface {
 	AccountID() string
-	Region() string
 	Details() string
+	Provider() string
+	Region() string
+	SubscriptionTier() defangv1.SubscriptionTier
```

Comment:
I see why this was added here, but wonder if this confuses the account (aws/do/gcp/…) with the Defang plan/tier.